plugin/trino-iceberg/src/main/java/io/trino/plugin/iceberg/IcebergUtil.java

```diff
@@ -310,11 +316,20 @@ public static List<Integer> buildPath(Map<Integer, Integer> indexParents, int fi
     public static Map<String, Object> getIcebergTableProperties(BaseTable icebergTable)
     {
         ImmutableMap.Builder<String, Object> properties = ImmutableMap.builder();
+        IcebergFileFormat fileFormat = getFileFormat(icebergTable);
         properties.put(FILE_FORMAT_PROPERTY, getFileFormat(icebergTable));
```

Comment:
duplicate call to `getFileFormat`, you can pass the instance from the previous line here instead

Reply:
done